x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/io/stream/PlanStreamInputTests.java

```diff
@@ -86,4 +96,54 @@ List<Long> randomLongsListOfSize(int size) {
         }
         return longs.stream().toList();
     }
+
+    public void testSourceSerialization() {
+        Function<String, String> queryFn = d -> d
```

Comment:
super-duper nit: since as a reader I don't have much context when I start reading this test, using an expressive lambda argument name could help a little.
```suggestion
        Function<String, String> queryFn = lineDelimiter -> lineDelimiter
```